chef/cookbooks/crowbar/recipes/crowbar-db-dump.rb

```diff
@@ -106,6 +110,10 @@
       done
     EOF
   end
+  # Deletes postgres data dir in non-HA scenario
+  directory "/var/lib/pgsql/data" do
+    action :delete
```

Comment:
> If you do have serious concerns

As said, I am undecided on this. But as we now already have already 2.5 people leaning towards keeping it, let's keep it and just rename the directory after creating the dump and stopping postgresql. In the end we just need to make sure the the new postgresql has a clean data directory when reapplying the database barclamp.
"Better be safe than sorry" I guess.

We can still tell the user to remove it/move it elsewhere if the isn't enough diskspace when we try to reload the dump.